### ISSUE TITLE
SpreadsheetRow.shouldComponentUpdate(): Returned undefined instead of a boolean value

### ISSUE THREAD

**Issue by acailly** [denisraslov/react-spreadsheet-grid#18]
* `react-spreadsheet-grid` version: 1.3.1
* `Operating System`: Windows 7 64bits

## Relevant code or config

```javascript
<Grid
        rows={this.state.rows}
        columns={this.columns}
        getRowKey={row => row.id}
      />
```

## What happened:

```
Warning: SpreadsheetRow.shouldComponentUpdate(): Returned undefined instead of a boolean value. Make sure to return true or false.
```

## Problem description:

React warns about `SpreadsheetRow.shouldComponentUpdate` not returning true or false

## Suggested solution:

If `SpreadsheetRow.shouldComponentUpdate` returns undefined, return false or true

**Comment by denisraslov** [comment]
Hi!
Thank you, I will fix it in the next release.
Could you clarify what version of React do you use?

**Comment by acailly** [comment]
I'm using React 15.5.4

**Comment by cvinson** [comment]
Addressed in https://github.com/denisraslov/react-spreadsheet-grid/pull/21

**Comment by denisraslov** [comment]
Fixed in the **release 1.4.1**. 🎉
